lib/device.cpp

```diff
@@ -20,19 +18,16 @@ Device::Device(udev_device* device)
         d->subsystem = udev_device_get_subsystem(d->device);
         d->type = udev_device_get_devtype(d->device);
 
-
-      //  qDebug() << QString("%1 - %2 - %3").arg(d->node).arg(d->subsystem).arg(d->type);
+        // qDebug() << QString("%1 - %2 - %3").arg(d->node).arg(d->subsystem).arg(d->type);
+        qDebug() << "TYPE :" << Device::deviceType(); // Test funtion
```

Comment:
no need to specify "Device::". The base class function will be automatically called